manim/mobject/svg/brace.py

```diff
@@ -228,3 +228,76 @@ def __init__(
             line_vector = np.array(point_2) - np.array(point_1)
             direction = np.array([line_vector[1], -line_vector[0], 0])
         Brace.__init__(self, Line(point_1, point_2), direction=direction, **kwargs)
+
+
+class ArcBrace(Brace):
+    """Creates a Brace that wraps around an Arc.
+
+    The direction parameter allows the brace to be applied
+    from outside or inside the Arc.
+
+    .. warning::
+        The ArcBraces are smaller for Arcs with smaller radii.
+
+    .. note::
+        The ArcBrace is initially a vertical Brace that takes the dimensions of a line
+        with the length of the Arc but scaled down to match the start and end angles,
+        then it is applied an exponential function after it is shifted
+        according to the radii of the arc.
+        The scale is not applied for arcs with radii smaller than 1 as it will
+        experience overscaling which is unideal.
+
+    Parameters
+    ----------
+    arc :
+        The Arc to which the Brace wraps around.
+    direction :
+        The direction from which the brace faces the Arc.
+        LEFT for inside the Arc, and RIGHT for the outside.
+
+    Example
+    -------
+        .. manim:: ArcBraceExample
+            :save_last_frame:
+            :ref_classes: Arc
+
+            class ArcBraceExample(Scene):
+                def construct(self):
+                    arc_1 = Arc(radius=1.5,start_angle=0,angle=2*PI/3).set_color(RED)
+                    brace_1 = ArcBrace(arc_1,LEFT)
+                    group_1 = VGroup(arc_1,brace_1)
+
+                    arc_2 = Arc(radius=3,start_angle=0,angle=5*PI/6).set_color(YELLOW)
+                    brace_2 = ArcBrace(arc_2)
+                    group_2 = VGroup(arc_2,brace_2)
+
+                    arc_3 = Arc(radius=0.5,start_angle=-0,angle=PI).set_color(BLUE)
+                    brace_3 = ArcBrace(arc_3)
+                    group_3 = VGroup(arc_3,brace_3)
+
+                    arc_4 = Arc(radius=0.2,start_angle=0,angle=3*PI/2).set_color(GREEN)
+                    brace_4 = ArcBrace(arc_4)
+                    group_4 = VGroup(arc_4,brace_4)
+
+                    arc_group = VGroup(group_1, group_2, group_3, group_4).arrange_in_grid(buff=1.5)
+                    self.add(arc_group.center())
+
+    """
+
+    def __init__(
+        self, arc=Arc(start_angle=-1, angle=2, radius=1), direction=RIGHT, **kwargs
+    ):
+        arc_center = arc.get_arc_center()
+        arc_start_angle = arc.start_angle
+        arc_end_angle = arc_start_angle + arc.angle
+        arc_radius = arc.radius
```

Comment:
Why not just use the properties of the arc instead of defining these new variables? For `end_angle`, it might be wise to add this attribute to `Arc` itself

Reply:
That's true.. i'll go ahead and fix that..